docs/source/development/linting.md

```diff
@@ -0,0 +1,84 @@
+# Linting
+
+## Introduction
+Lint, or a linter, is a static code analysis tool used to flag programming errors, bugs, stylistic errors and suspicious constructs.
+As a project grows and goes through various stages of development, it becomes important to maintain the code quality. Linting makes
+your code more readable, easy to debug and maintain, and stylistically consistent.
+## Set up linting tools
+There are a variety of linting tools available to use
+with your Kedro projects. This guide shows you how to use [`black`](https://github.com/psf/black), [`flake8`](https://gitlab.com/pycqa/flake8), and [`isort`](https://github.com/PyCQA/isort) to lint your Kedro projects.
+- **`black`** is a [PEP 8](https://peps.python.org/pep-0008/) compliant opinionated Python code formatter. Read the full documentation [here](https://black.readthedocs.io/en/stable/).
+- **`flake8`** is a wrapper around [`pep8`](https://pypi.org/project/pep8/), [`pyflakes`](https://pypi.org/project/pyflakes/), and [`mccabe`](https://pypi.org/project/mccabe/). Read the full documentation [here](https://flake8.pycqa.org/en/latest/).
+- **`isort`** is a Python library used to sort imports alphabetically. Read the full documentation [here](https://pycqa.github.io/isort/).
```

Comment:
I think the introduction to `black`, `isort` and `flake8` could be a bit longer, maybe with an example each of what they can do for the reader.